packages/clerk-js/src/ui/components/Plans/PlanDetails.tsx

```diff
@@ -38,18 +38,19 @@ export const PlanDetails = (props: __internal_PlanDetailsProps) => {
 
 const BodyFiller = ({ children }: { children: React.ReactNode }) => {
   return (
-    <Drawer.Body>
-      <Flex
-        align={'center'}
-        justify={'center'}
-        sx={t => ({
-          height: '100%',
-          padding: t.space.$4,
-          fontSize: t.fontSizes.$md,
-        })}
-      >
-        {children}
-      </Flex>
+    <Drawer.Body
+      sx={t => ({
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        flex: 1,
+        overflowY: 'auto',
```

Comment:
body already handles a majority of these.

https://github.com/clerk/javascript/blob/79b4f032505314fce9fe88ed0929ed30742b393a/packages/clerk-js/src/ui/elements/Drawer.tsx#L351-L355